o2-full-system-test.sh

```diff
@@ -24,11 +24,16 @@ $O2_ROOT/prodtests/full_system_test_ci_extra_tests.sh
 popd
 rm -Rf $BUILDDIR/full-system-test-sim
 
+# during build, the Python paths are not setup (in contrast to a full module load env)
+# python is needed by the O2DPG scheduler
+export PYTHONPATH=${PYTHONPATH}:${PYTHON_MODULES_ROOT}/share/python-modules/lib/python/site-packages
+
```

Comment:
Would it be better to use the following in the frontmatter instead? Then the path is set for this recipe and for anything that may end up depending on this recipe, I think.
```yaml
append_path:
  PYTHONPATH: "$PYTHON_MODULES_ROOT/share/python-modules/lib/python/site-packages"
```

Reply:
I tried to put:
```
build_requires:
  - Python
  - Python-modules
append_path:
  PYTHONPATH: "$PYTHON_MODULES_ROOT/share/python-modules/lib/python/site-packages"
```
but it is not working correctly.

Reply:
append_path is for the dependent of the recipe, not for the build itself, so it needs to be there in the `Python-modules` recipe, not in this one.

Reply:
The issue is the LHAPDF recipe, which resets the PYTHONPATH.